src/LighthouseServiceProvider.php

```diff
@@ -38,25 +36,33 @@ class LighthouseServiceProvider extends ServiceProvider
     /**
      * Bootstrap any application services.
      *
+     * @param  \Illuminate\Validation\Factory  $validationFactory
      * @return void
      */
-    public function boot(): void
+    public function boot(ValidationFactory $validationFactory): void
     {
-        $this->mergeConfigFrom(__DIR__.'/../../config/config.php', 'lighthouse');
+        $this->mergeConfigFrom(__DIR__.'/../config/config.php', 'lighthouse');
 
         $this->publishes([
-            __DIR__.'/../../config/config.php' => config_path('lighthouse.php'),
+            __DIR__.'/../config/config.php' => config_path('lighthouse.php'),
         ], 'config');
 
         $this->publishes([
-            __DIR__.'/../../assets/default-schema.graphql' => config('lighthouse.schema.register'),
+            __DIR__.'/../assets/default-schema.graphql' => config('lighthouse.schema.register'),
```

Comment:
Remove `config` helper.

Reply:
In favour of?

Reply:
Take in the config repository in the constructor.

Reply:
Done